### ISSUE TITLE
can you share hyper-parameter ?

### ISSUE THREAD

**Issue by taki0112** [Yijunmaverick/CartoonGAN-Test-Pytorch-Torch#5]
Hi
I'm interested your paper
So I'm reproducing the CartoonGAN

However, I don't know hyper-paramter in detail.. like weight initialization.. (xavier ? gaussian?)
I want to know hyper-parameter information
1. Optimizer (if Adam, what `beta1`, beta2` ?)
2. Weight initialization
3. Use l2 regularization (weight decay) ?
4. Data augmentation
5. Normalization function in discriminator (batch norm? layer norm?)

Thank you !

**Comment by Yijunmaverick** [comment]
@taki0112 Sorry, this is not my work. You can ask authors for details.

**Comment by taki0112** [comment]
@Yijunmaverick 
I have a question about vgg's conv4_4 feature map.
Is this a feature map after relu? Or is it a feature map before relu?

**Comment by FlyingGoblin** [comment]
@Yijunmaverick  before relu.
I am sorry I didn't clean up my code because of the graduation. I will try my best to upload my code in this month.
For the photo, all the images in our paper are from CycleGAN: https://github.com/junyanz/pytorch-CycleGAN-and-pix2pix (XXX2photo).
Because all the cartoon images are from films, I don't think I have the right to upload them. I don't know if there will be any copyright problem， I am sorry.

**Comment by Yijunmaverick** [comment]
@taki0112 Check the author's reply above

**Comment by FlyingGoblin** [comment]
@taki0112 the code: https://github.com/FlyingGoblin/CartoonGAN, sorry for the delay
